data/mil2525d/core_data/gdbs/README.md

```diff
@@ -39,32 +39,41 @@ The following modifier fields are supported in [the current Pro Stylx and Runtim
-| quantity | C | C | Quantity | |
-| reinforced | F | F | Reinforced or Reduced | |
-| reliability | J | J | Evaluation Rating | Reliability rating is first character of Evaluation Rating (J) field. |
-| sigintmobility | R2 | R2 | SIGINT Mobility Indicator | |
-| signatureequipment | L | L | Signature Equipment | |
-| speed | Z | Z | Speed | |
-| staffcomment | G | G | Staff Comments | |
-| type | V | V | Type | |
-| uniquedesignation | T | T | Unique Designation | |
-| uniquedesignation2 | T | | Unique Designation 2 | Used as an additional field for Fire Support Lines tactical graphics. |
-| x | Y | Y | Location | Longitude in degrees. |
-| y | Y | Y | Location | Latitude in degrees. |
-| z | X | X | Altitude/Depth | |
-| zmax | X | X | Altitude/Depth | Maximum altitude for aviation tactical graphics. |
-| zmin | X | X | Altitude/Depth | Minimum altitude for aviation tactical graphics. |
+| Attribute Name | 2525 D Field ID | Field Title | Notes |
+| ------------- | --- | ----------------- | ----- |
+| additionalinformation | H | Additional Information | |
+| additionalinformation2 | H | Additional Information 2 | Used in the Convoys and Airspace Coordination Area tactical graphics. |
+| azimuth | AN | Azimuth | |
+| combateffectiveness | K | Combat Effectiveness | |
+| commonidentifier | AF | Common Identifier | |
+| country | AS | Country Code | |
+| credibility | J | Evaluation Rating | Credibility rating is second character of Evaluation Rating (J) field. |
+| datetimeexpired | W | Date-Time Group (DTG) | Second half of Date-Time Group (DTG) (W) field. |
+| datetimevalid | W | Date-Time Group (DTG) | First half of Date-Time Group (DTG) (W) field. |
+| distance | AM | Distance | |
+| distance1 | AM1 | Distance 2 | |
+| direction | Q | Direction of Movement Indicator | |
+| equipmentteardowntime | AE | Equipment Teardown Time | |
+| higherformation | M | Higher Formation | |
+| hostile | N | Hostile | |
+| idmode | P | IFF/SIF/AIS | |
+| platformtype | AD | Platform Type | |
+| quantity | C | Quantity | |
+| reinforced | F | Reinforced or Reduced | |
+| reliability | J | Evaluation Rating | Reliability rating is first character of Evaluation Rating (J) field. |
+| sigintmobility | R2 | SIGINT Mobility Indicator | |
+| signatureequipment | L | Signature Equipment | |
+| speed | Z | Speed | |
+| staffcomment | G | Staff Comments | |
+| targetdesignator | AP | Target Designator |
+| type | V | Type | |
+| uniquedesignation | T | Unique Designation | |
+| uniquedesignation2 | T1 | Unique Designation 2 | Used as an additional field for Fire Support Lines tactical graphics. |
+| x | Y | Location | Longitude in degrees. |
+| x1 | Y1 | Location | Longitude in degrees 2. |
+| y | Y | Location | Latitude in degrees. |
+| y1 | Y1 | Location | Latitude in degrees 2. |
+| z | X | Altitude/Depth | |
+| z1 | X1 | Altitude/Depth 2 | |
+| zmax | X | Altitude/Depth | Maximum altitude for aviation tactical graphics. |
```

Comment:
Do we actually have fields called "zmax" and "zmin"? these fields might be in error (and description added/moved to X/X1 row)